templates/iot/api/010-CustomResourceDefinition-iot-project.yaml

```diff
@@ -32,9 +32,9 @@ spec:
     type: boolean
     description: If TLS is enabled
     JSONPath: .status.downstreamEndpoint.tls
-  - name: Ready
-    type: boolean
-    description: If the IoT project is ready
-    JSONPath: .status.isReady
```

Comment:
Is it intentional to not keep isready? I think it could be useful.

Reply:
Yes. I switch to useing "Phase", which seems to be the way Kubernetes does things. The problem with "ready" being a boolean was, that OLM didn't properly understand the boolean type.

It looks to me as if Kubernetes uses Phase/PhaseReason + Conditions to show that something is ready.

Reply:
Good point. Should IsReady be removed from the model as well?

It could be worth standardizing on the 'conditions' array pattern for readiness instead, probably before API is declared beta.

Reply:
Something like.... https://github.com/strimzi/strimzi-kafka-operator/blob/master/install/cluster-operator/040-Crd-kafka.yaml#L3538

Reply:
> Good point. Should IsReady be removed from the model as well?
> 
> It could be worth standardizing on the 'conditions' array pattern for readiness instead, probably before API is declared beta.

Yes and no :) For example, for the "oc get" command it is easier to work with a non-array field. On the other hand conditions allow a more complex description of the current status. The API model currently uses the v1.Conditions from Kubernetes, so I think that aligns well with the Kubernetes world. I think having both Phase + Conditions is ok. Only having a third (boolean) seemed to be bit too much :)